Essentials/src/messages.properties

```diff
@@ -449,6 +449,7 @@ replyLastRecipientEnabledFor=\u00a76Replying to last message recipient \u00a7cen
 requestAccepted=\u00a76Teleport request accepted.
 requestAcceptedFrom=\u00a7c{0} \u00a76accepted your teleport request.
 requestDenied=\u00a76Teleport request denied.
+requestDeniedCooldown=\u00a76{0} has a teleport cooldown.
```

Comment:
And I added the message (at least I think we're supposed to add it here). I'm open to suggestions for a different message.

Reply:
I'd change this to `\u00a76{0} is currently on a teleport cooldown.` as that's a bit clearer.